soloco_perception:
  human_tracker:
    ros_parameters:
      # Subscriber topics
      detected_obj_topic: "zed2/zed_node/obj_det/objects"
      # Publisher topics
      human_track_topic: "human/interpolated_history"
      pose_marker_topic: "visualization/human_tracks"
      bounding_box_topic: "visualization/human_bounding_boxes"
      pub_frame_id: "locobot/odom"
      # Parameters
      interp_interval: 0.4      # time interval between interpolation points [s]
      max_history_length: 7     # maximum number of history points (without current)
      max_num_agents: 5         # currently not used
      track_timeout: 0.15       # maximum time to keep old tracks before deleting it [s]
      delay_tolerance: 0.05     # delay tolerance for track timeout [s]
      pruning_rate: 25          # Rate at which to check whether a track is old [Hz]
      visualize_bbox: true      # Whether to visualize bounding boxes for the current human poses